spec/unit/resources/osl_synapse_admin_spec.rb

```diff
@@ -0,0 +1,31 @@
+#
+# Cookbook:: osl-matrix
+# Spec:: osl_element
+#
+# Copyright:: 2023-2025, Oregon State University
+#
+
+require_relative '../../spec_helper'
+
+describe 'osl-matrix-test::synapse-admin' do
+  cached(:subject) { chef_run }
+  platform 'almalinux', '8'
+  step_into :osl_synapse_admin
```

Comment:
Please switch to doing the tests using [this method](https://docs.osuosl.org/config-management/chef/testing/chefspec.html#platforms-iterated-over-a-loop).